app/controlplane/internal/biz/apitoken.go

```diff
@@ -72,6 +72,8 @@ func NewAPITokenUseCase(apiTokenRepo APITokenRepo, conf *conf.Auth, authzE *auth
 		logger:       servicelogger.ScopedHelper(logger, "biz/APITokenUseCase"),
 		enforcer:     authzE,
 		DefaultAuthzPolicies: []*authz.Policy{
+			// Add permissions to workflow run
+			authz.PolicyWorkflowRunList,
```

Comment:
we are going to need describe too. The one you provided only allow to list runs not describe them.

Reply:
That was going to be my follow up question. Adding it.